com.github.Resticity.yml

```diff
@@ -0,0 +1,48 @@
+id: com.github.Resticity
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+
+
+command: resticity
+finish-args:
+  - --share=ipc # Share IPC namespace with the host (necessary for X11).
+  - --socket=x11
+  - --device=dri # OpenGL rendering support.
+  - --talk-name=org.freedesktop.Notifications
+  - --filesystem=home
+  - --filesystem=host
+  - --socket=wayland
+  - --share=network
+  - --talk-name=org.kde.StatusNotifierWatcher
+
+build-options:
+  build-args:
+    - --share=network
+  env:
+    - GOROOT=/usr/lib/sdk/golang
+    - GOBIN=/app/go/bin
+    - CGO_ENABLED=0
+
+
+modules:
+  - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
```

Comment:
and where are the shared modules?

the shared modules doc say to add theme with git submodule, which isn't done here.